ucs/ports/inbound/file_service.py

```diff
@@ -16,38 +16,38 @@
 """Interfaces for the main upload handling logic of this service."""
 
 
-from typing import Iterable, Protocol, Sequence
+from abc import ABC, abstractmethod
+from typing import Iterable, Sequence
 
 from ucs.core import models
 
 UPDATABLE_METADATA_FIELDS = {"status"}
 
 
-class FileUnkownError(RuntimeError):
-    """Thrown when a file with the given ID is not known."""
-
-    def __init__(self, *, file_id: str):
-        self.file_id = file_id
-        message = f"The file with ID {file_id} is unkown."
-        super().__init__(message)
-
+class FileMetadataServicePort(ABC):
+    """Interface of a service handling file metata."""
 
-class InvalidFileMetadatUpdateError(RuntimeError):
-    """Thrown when trying to update a metadata field of a file that is not allowed to
-    change (i.e. not in the UPDATABLE_METADATA_FIELDS set)."""
+    class FileUnkownError(RuntimeError):
+        """Thrown when a file with the given ID is not known."""
 
-    def __init__(self, *, file_id: str, invalid_fields: Iterable[str]):
-        self.file_id = file_id
-        message = (
-            f"Following fields for the with ID {file_id} cannot be updated: "
-            + ", ".join(invalid_fields)
-        )
-        super().__init__(message)
+        def __init__(self, *, file_id: str):
+            self.file_id = file_id
+            message = f"The file with ID {file_id} is unkown."
+            super().__init__(message)
 
+    class InvalidFileMetadatUpdateError(RuntimeError):
```

Comment:
```suggestion
    class InvalidFileMetadataUpdateError(RuntimeError):
```
Needs to be changed at exception site too